src/components/browse/components/SelectedItemsController.js

```diff
@@ -129,42 +104,38 @@ export class SelectedItemsController extends React.PureComponent {
-            // Also add the popLink/target=_blank functionality to links
-            // Remove lab.display_title and type columns on selection
-            const newColumnExtensionMap = _.clone(originalColumnExtensionMap);
-            newColumnExtensionMap.display_title = {
-                ...newColumnExtensionMap.display_title,
-                'minColumnWidth' : (originalColumnExtensionMap.display_title.minColumnWidth || 100) + 20,
-                'render' : (result, parentProps) => {
-                    const { selectedItems } = this.state;
-                    const { rowNumber, detailOpen, toggleDetailOpen, href, context } = parentProps;
-                    return (
-                        <DisplayTitleColumnWrapper {...{ result, href, context, rowNumber, detailOpen, toggleDetailOpen }}>
-                            <SelectionItemCheckbox {...{ selectedItems, isMultiSelect }} handleSelectItem={this.handleSelectItem} />
-                            <DisplayTitleColumnDefault />
-                        </DisplayTitleColumnWrapper>
-                    );
-                }
-            };
-            return newColumnExtensionMap;
-        }
-        return originalColumnExtensionMap;
+        // Render out button and add to title render output for "Select" if we have a 'selection' currentAction.
+        // Also add the popLink/target=_blank functionality to links
+        // Remove lab.display_title and type columns on selection
+        const newColumnExtensionMap = _.clone(originalColumnExtensionMap);
+        newColumnExtensionMap.display_title = {
+            ...newColumnExtensionMap.display_title,
+            'minColumnWidth' : (originalColumnExtensionMap.display_title.minColumnWidth || 100) + 20,
+            'render' : (result, parentProps) => {
+                const { selectedItems } = this.state;
+                const { rowNumber, detailOpen, toggleDetailOpen, href, context } = parentProps;
+                return (
+                    <DisplayTitleColumnWrapper {...{ result, href, context, rowNumber, detailOpen, toggleDetailOpen }}>
+                        <SelectionItemCheckbox {...{ selectedItems }} onSelectItem={this.handleSelectItem}
+                            isMultiSelect={currentAction === 'multiselect'} />
+                        <DisplayTitleColumnDefault />
+                    </DisplayTitleColumnWrapper>
+                );
+            }
+        };
+        return newColumnExtensionMap;
     }
 
     render(){
-        const { children, ...propsToPass } = this.props;
+        const { children, ...remainingProps } = this.props;
         const { selectedItems } = this.state;
-
-        _.extend(propsToPass, {
+        const propsToPass = {
+            ...remainingProps,
             selectedItems,
             columnExtensionMap      : this.columnExtensionMapWithSelectButton(),
-            onSelectItem            : this.handleSelectItemClick,
```

Comment:
This was a typo -- corrected from `this.handleSelectItemClick` to `this.handleSelectItem`